src/com/lftechnology/java/training/srijan/basics/coreapp/ui/EmployeeView.java

```diff
@@ -0,0 +1,397 @@
+			e.setFullName(fullname);
+			LOGGER.log(Level.INFO, "Enter Department*:");
+			this.department = input.nextLine();
+			e.setDepartment(department);
+			LOGGER.log(Level.INFO, "Enter Address:");
+			this.address = input.nextLine();
+			e.setAddress(address);
+			LOGGER.log(Level.INFO, "Enter password*");
+			this.password = input.nextLine();
+			e.setPassword(password);
+			LOGGER.log(
+				Level.INFO,
+				"Enter User Role*:\n1.ADMIN\n2.NORMAL\n3.Default: NORMAL");
+			this.choiceUserRole = input.nextInt();
+			switch (choiceUserRole) {
+			case 1:
+				e.setUserRole(UserRole.ADMIN);
+				break;
+			case 2:
+				e.setUserRole(UserRole.NORMAL);
+				break;
+			default:
+				LOGGER.log(
+					Level.INFO,
+					"please choose right option,Default Normal user assigned");
+				e.setUserRole(UserRole.NORMAL);
+				break;
+			}
+			LOGGER.log(
+				Level.INFO,
+				"User status:\n0.Not terminated\n1.Terminated\n2.Default: Not Terminated");
+			userStatus = input.nextInt();
+			switch (userStatus) {
+			case 0:
+				e.setIsTerminated(false);
+				break;
+			case 1:
+				e.setIsTerminated(true);
+				break;
+			default:
+				LOGGER.log(
+					Level.INFO,
+					"Wrong option, User has been default assigned to Not Terminated");
+				e.setIsTerminated(false);
+				break;
+			}
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception", ex);
+		}
+		return e;
+	}
+
+	/**
+	 * <p>Calls different user methods depending upon user choice</p>
+	 * 
+	 * @param normalUserChoice
+	 * @param userName
+	 * @author srijan
+	 */
+	public void normalUserOptionHandler(int normalUserChoice, Employee em,Scanner input) {
+
+		switch (normalUserChoice) {
+		case 1:
+			employees = employeeControl.viewAllUser(em);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			break;
+		case 2:
+			employee = employeeControl.findUserByUserName(em);
+			LOGGER.log(Level.INFO, "{0}", employee);
+			Employee changedProfileDetails = changeProfile(employee,input);
+			Boolean status =
+				employeeControl.updateProfile(changedProfileDetails);
+			if (status.equals(true)) {
+				LOGGER.log(Level.INFO, "Update Successful");
+			}
+			break;
+		case 3:
+			exit();
+			break;
+		default:
+			LOGGER.log(Level.INFO,
+				"Please choose option properly,Default view all user block");
+			employees = employeeControl.viewAllUser(em);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			break;
+		}
+	}
+
+	/**
+	 * <p>edit and terminate user</p>
+	 * 
+	 * @return
+	 * @author srijan
+	 */
+	public int editAndTerminateEmployee(Scanner input) {
+
+		int employeeIdToEdit = 0;
+		try {
+			input = new Scanner(System.in);
+			employeeIdToEdit = input.nextInt();
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception", ex);
+		}
+		return employeeIdToEdit;
+	}
+
+	/**
+	 * <p>Change profile of an user</p>
+	 * 
+	 * @param emp
+	 * @return
+	 * @author srijan
+	 */
+	public Employee changeProfile(Employee emp,Scanner input) {
+
+		try {
+			input = new Scanner(System.in);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "input". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)